ui/app.js

```diff
@@ -3878,6 +3982,487 @@ $(document).ready(() => {
+          document.documentElement.style.setProperty(
+            "--color-9", /* backround */
+            "#6E707C"
+          );
+          document.documentElement.style.setProperty(
+            "--color-10", /* Refresh button */
+            "#8F741B"
+          );
+          $(".badge-logo").attr("src", "img/LSSD.webp");
+          $(".header-title").html("LOS SANTOS SHERIFF DEPARTMENT");
+          $(".bolo-nav-item").html("BOLOs");
+          $(".bolos-search-title").html("Bolos");
+          $("#bolos-search-input").attr("placeholder", "Search Bolo...");
+          $(".manage-bolos-title").html("Manage Bolo");
+          $(".manage-bolos-editing-title").html(
+            "You are currently creating a new BOLO"
+          );
+          $(".boloplate-title").html("Plate");
+          $(".boloowner-title").html("Owner");
+          $(".boloindividual-title").html("Individual");
+          $("#boloplate").attr("placeholder", "Place plate here...");
+          $("#bolodetail").attr(
+            "placeholder",
+            "Bolo detail goes here..."
+          );
+          $("#boloowner").attr(
+            "placeholder",
+            "Place vehicle owner here..."
+          );
+          $("#boloindividual").attr(
+            "placeholder",
+            "Place invidivual here..."
+          );
+          $("#home-warrants-container").fadeIn(0);
+          $("#home-reports-container").fadeOut(0);
+          //$(".quote-span").html("TUCKER MALD, BEST MALD");
+          $(".incidents-nav-item").show();
+          $(".bolo-nav-item").show();
+          $(".dmv-nav-item").show();
+          $(".weapons-nav-item").show()
+          $(".cams-nav-item").show();
+          $(".map-nav-item").show();
+          $(".dispatch-title-ofsomesort").html("Dispatch");
+          $(".dispatch-comms-container").fadeIn(0);
+          $(".manage-profile-name-input-1").attr("readonly", true);
+          $(".manage-profile-name-input-2").attr("readonly", true);
+          $("#reports-officers-involved-tag-title").html(
+            "Officers Involved"
+          );
+          $(".roster-iframe").attr("src", rosterLink);
+  
+          $(".manage-profile-save").css("display", "block");
+          $(".manage-profile-editing-title").css("display", "block");
+          $(".manage-incidents-create").css("display", "block");
+          $(".manage-incidents-save").css("display", "block");
+          $(".manage-incidents-editing-title").css("display", "block");
+          $(".manage-reports-new").css("display", "block");
+          $(".manage-reports-save").css("display", "block");
+          $(".manage-reports-editing-title").css("display", "block");
+          $(".vehicle-information-save").css("display", "block");
+          $(".vehicle-information-title").css("margin-right", "0px").css("width", "81%");
+          $(".manage-incidents-title ").css("margin-right", "0px")
+          $(".manage-reports-title").css("margin-right", "0px").css("width", "66%");

```

Comment:
nit: There is a lot of duplicated code here, it might be better to use a more generic solution.

Possible solution is to make a function that defines these once and then pass in an override variable that will be used
Then you could create an object that defines the unique values to use for each one.

Ex:
```
const customThemes = {
  lspd: {
     color1: #1E3955,
     color2: #213F5F,
    .... rest of the styles
  },
  bcso: {
   .... rest of the styles
  },
... other themes
};


function applyCustomTheme(theme) {
   document.documentElement.style.setProperty(
            "--color-1", /* Panels */
            theme.color1
          );
          document.documentElement.style.setProperty(
            "--color-2", /* Clock */
            theme.color2
          );
.... rest of the custom styles applied from theme
}



Then in each section where you are applying the custom theme you could do something like 
if (sentJob == "police") {
 applyCustomTheme(customThemes.lspd)
} elseif(sentJob == "bcso") {
 applyCustomTheme(customThemes.bcso)
 } 
 ...

```

Reply:
thanks, I have been working on cleaning up the code a bit so it wasn't as bloated this helps ALOT. I'm not too good at JS. I will look at implementing this today.